### ISSUE TITLE
"loss: nan"   when training faster_rcnn_r50_fpn_giou_1x_coco.py

### ISSUE THREAD

**Issue by WHAOOOOO** [open-mmlab/mmdetection#3291]
When training config of “faster_rcnn_r50_fpn_giou_1x_coco.py”, the loss is nan.

my mmdet version is 2.2.0.
ubuntu16.04; cuda10.2; pytorch1.5; python3.7.

**Comment by zhaoyang97** [comment]
I had the same problem. Would u please tell me how u solve it？
